app/views/tweets/edit.html.erb

```diff
@@ -1,3 +1,10 @@
+<% if @tweet.errors.full_messages.any? %>
+  <h4>Errors</h4>
+  <% @tweet.errors.full_messages.each do |error_message| %>
+    <%= error_message if @tweet.errors.full_messages.first == error_message %> <br />
```

Comment:
Hey, I'm confused by this conditional.  It would only show the first full message associated with an error.  Why not show them all?